apps/qubit/modules/informationobject/templates/deleteFindingAidSuccess.php

```diff
@@ -0,0 +1,24 @@
+<?php decorate_with('layout_1col.php') ?>
+
+<?php slot('title') ?>
+  <h1><?php echo __('Are you sure you want to delete the finding aid?') ?></h1>
+<?php end_slot() ?>
+
+<?php slot('content') ?>
+
+  <?php echo $form->renderGlobalErrors() ?>
+
+  <?php echo $form->renderFormTag(url_for(array($resource, 'module' => 'informationobject', 'action' => 'deleteFindingAid')), array('method' => 'delete')) ?>
+
+    <?php echo $form->renderHiddenFields() ?>
+
+    <section class="actions">
+      <ul>
+        <li><?php echo link_to(__('Cancel'), array($resource, 'module' => 'informationobject'), array('class' => 'c-btn')) ?></li>
+        <li><input class="c-btn c-btn-delete" type="submit" value="<?php echo __('Delete') ?>"/></li>
+      </ul>
+    </section>
+
+  </form>
+
+<?php end_slot() ?>
```

Comment:
Could you show the title of the corresponding information object and perhaps the filename or public path? The idea is to provide some context so we can make easier for the user to decide if it really should be removed.